javascript-timers/interval.js

```diff
@@ -10,6 +10,8 @@ function countDownEarth() {
   } else if ($h1.textContent === '1') {
     $h1.textContent = '~Earth Beeeelooowww Us~';
   }
+  var countDownId = setInterval(countDownEarth, 1000);
```

Comment:
Close, but that variable should be set on line 17. On line 13 you unnecessarily are introducing a whole new setInterval function call inside of the else if. 

So remove this line: `var countDownId = setInterval(countDownEarth, 1000);`